extension/tamper.css

```diff
@@ -31,6 +31,7 @@ h1 {
 label {
   display: block;
   padding: 5px 0;
+  color: blue;
```

Comment:
tried it out, and chrome.devtools.panels.themeName seems available on my current version of chrome (Version 57.0.2987.146 (64-bit)), so probably we just need to add some code to JS that does:
`document.body.className = chrome.devtools.panels.themeName;`
and then have some code that does:
`
  body.dark label {
    color: #ddd;
  }
`